lib/Alien/Build/Util.pm

```diff
@@ -76,7 +76,26 @@ sub _mirror
         { unlink "$dst" }
         my $target = readlink "$src";
         Alien::Build->log("ln -s $target $dst") if $opt->{verbose};
+        if (path($target)->is_relative) {
+          my $nativesymlink =
+                   (($^O eq "msys" && defined $ENV{MSYS} && $ENV{MSYS} eq "winsymlinks:nativestrict")
+                || ($^O eq "cygwin" && defined $ENV{CYGWIN} && $ENV{CYGWIN} eq "winsymlinks:nativestrict"));
```

Comment:
I don't think we should be checking the `MSYS` or `CYGWIN` variables for this tbh, but note that CYGWIN at least is space separated set of options:
https://cygwin.com/cygwin-ug-net/using-cygwinenv.html

Reply:
> note that CYGWIN at least is space separated set of options

Thanks, but `winsymlinks:nativestrict` is then a single option and the colon is part of the option, not a separator. I assume the MSYS variable behaves the same way. I will update the code to handle the case with more than one option.